libs/openFrameworks/video/ofVideoPlayer.cpp

```diff
@@ -77,7 +77,7 @@ bool ofVideoPlayer::load(string name){
         if(bUseTexture){
         	if(player->getTexturePtr()==nullptr){
 				if(tex.empty()) {
-					tex.resize(max(player->getPixels().getNumPlanes(),1));
```

Comment:
Nice!  I'd suggest prepending this with `std::max(` to be explicit.  Also, this should be tested on windows (Appveyor I guess) since std::min / std::max have windows problems ...